turn/server.go

```diff
@@ -95,6 +110,26 @@ func generator(conf config.Config) turn.RelayAddressGenerator {
 	return &RelayAddressGeneratorNone{}
 }
 
+func (a *Server) AcceptAccounts(client, host *TurnAccount) error {
+	client.Username = client.Id.String()
+	client.Credential = util.RandString(20)
+	a.Allow(client.Username, client.Credential, client.IP)
```

Comment:
This seems a little weird, 2 users with the same username but different credentials, is this valid?

Reply:
Oups, it's a mistake.

Reply:
`client` and `host` use their own Id for username.

The `AcceptAccounts` use now a variadic argument and accept in a loop.

Is it correct for you, @jmattheis ?

Reply:
Yeah, maybe we could remove the AcceptAccounts / Revoke methods & turn.account and just pass the username & password like it is done in the allow method (and call allow 2 times on session creation)? I feel like creating a struct (Account) that is only used in one method is overkill.